### ISSUE TITLE
BlazorServer App: Is there a way around StartUp.cs?

### ISSUE THREAD

**Issue by janschreier** [MackinnonBuck/blazor-playwright-example#1]
Hi Mackinnon,
thanks for providing this exampe (and the YouTube-video on Playwright!). 

I just re-created a StartUp.cs file to make my code run. Is there any way to keep the new format without StartUp.cs? I haven't found any method to define that default startup with the minimal API style but maybe you know of any?

Thanks for your help in advance!
Jan

**Comment by CameronMackenzie99** [comment]
Funnily enough I have just ran into the same problem! The only way around it I have found so far is this: https://github.com/aherrick/BlazorServerPlaywright/blob/main/BlazorServerPlaywright.Tests/Infrastructure/CustomWebApplicationFactory.cs where you pass a dummy server back as the host. I can't seem to find a good way to get around the loss of the BuildWebHost method. Any insight would be appreciated!
